includes/Modules/AdSense.php

```diff
@@ -427,9 +427,10 @@ protected function create_data_request( Data_Request $data ) {
 				};
 			case 'GET:account-url':
 				return function() {
-					$account_id = $this->get_data( 'account-id' );
+					$account_id   = $this->get_data( 'account-id' );
+					$current_user = wp_get_current_user();
 					if ( ! is_wp_error( $account_id ) && $account_id ) {
-						return sprintf( 'https://www.google.com/adsense/new/%s/home', $account_id );
+						return sprintf( 'https://www.google.com/adsense/new/u/%1$s/%2$s/home', $current_user->user_email, $account_id );
```

Comment:
@ryanwelcher The email needs to come from `\Google\Site_Kit\Core\Authentication\Profile` (`$this->authentication->profile()` in a module class) rather than the WP user's email. Even though these may be the same, the email from `Profile` is the email for their Google account they authenticated with Site Kit and the account that will have access to the service.

Reply:
Thanks @aaemnnosttv, I've made the update.